plugins/generic/externalFeed/classes/ExternalFeedDAO.inc.php

```diff
@@ -22,19 +22,33 @@ class ExternalFeedDAO extends DAO {
 	/**
 	 * Constructor
 	 */
-	function __construct($parentPluginName) {
+	public function __construct($parentPluginName) {
 		$this->parentPluginName = $parentPluginName;
 		parent::__construct();
 	}
 
+	/**
+	 * Instantiate a new data object.
+	 * @return ExternalFeed
+	 */
+	function newDataObject() {
+		import('plugins.generic.externalFeed.classes.ExternalFeed');
```

Comment:
It's likely that we're not doing this consistently, but it's better to use `$plugin->import` rather than hard-coding the canonical path to the plugin.